src/pages/UserProfile.js

```diff
@@ -6,19 +6,19 @@ import { PrimaryButton } from "../components/PrimaryButton";
 import { updateUserProfile } from "../store/actionCreators";
 
 function Profile() {
-  const { name, email, address, phoneNumber, _id, role } = useSelector(
-    state => state.user
-  );
+  const { name, email, address, phoneNumber, _id, role, photoUrl } =
+    useSelector(state => state.user);
   const dispatch = useDispatch();
 
   const [updateProfile, setUpdateProfile] = useState({
     _id: _id,
-    name: "",
+    name: name,
     email: email,
-    address: "",
-    phoneNumber: "",
-    photoUrl: null,
+    address: address,
```

Comment:
You can use only the key "address"

Reply:
Thank you for your feedback. I just fixed this using the keys only.

Thanks